sailor/assetcentral/system.py

```diff
@@ -193,7 +220,8 @@ class SystemSet(AssetcentralEntitySet):
     }
 
     def get_indicator_data(self, start: Union[str, pd.Timestamp, datetime.timestamp, datetime.date],
-                           end: Union[str, pd.Timestamp, datetime.timestamp, datetime.date], *,
+                           end: Union[str, pd.Timestamp, datetime.timestamp, datetime.date],
+                           indicator_set: IndicatorSet = None, *,
```

Comment:
Two remarks regarding `indicator_set`:

1) Please add the parameter also to the `get_indicator_data` function in `System` for consistency.
2) Please add the parameter to both docstrings. You can copy from Equipment I think